src/main/java/bio/terra/service/filedata/azure/util/BlobContainerCopier.java

```diff
@@ -118,17 +137,22 @@ public List<BlobCopySourceDestinationPair> getSourceDestinationPairs() {
     return sourceDestinationPairs;
   }
 
-  private BlobContainerCopySyncPoller beginCopyOperationUsingBlobSignedUrl() {
-    BlobUrlParts blobUrlParts = BlobUrlParts.parse(sourceBlobUrl);
+  private BlobContainerCopySyncPoller beginCopyOperationUsingUrl() {
+    String sourceBlobName;
+    if (sourceBlobUrl.startsWith("gs://")) {
```

Comment:
We really have a lot of `.startsWith("gs://")` calls now. Maybe it should be a function in `GcsUriUtils`, like `GcsUriUtils.isGsPath()`?

Reply:
Hmm, yeah, good point